### ISSUE TITLE
ECMPS 508 

### ISSUE THREAD

**Issue by vishnunavuluri** [US-EPA-CAMD/PI-1#2105]
RF-1a: Color Contrast is Insufficient 
from Report https://app.zenhub.com/files/287570343/c84cab03-f68f-421a-8476-705e237a8283/download
